test/test_scalar_parameters.py

```diff
@@ -55,42 +54,33 @@ def sinewave2D(comm):
     return (L, hm, top)
 
 
-@pytest.mark.skipif(
-    MPI.COMM_WORLD.Get_size() > 1,
-    reason="tests only serial functionalities, please execute with pytest")
-def test_rms_curvature(sinewave2D):
-    L, hm, top = sinewave2D
+def test_rms_curvature(comm_self):
```

Comment:
I do not understand while here you introduced comm_self. I guess you wanted to do a parallel test, where you need `comm`.

Reply:
That's right. I changed this to the parallel version.